.gitlab-ci.yml

```diff
@@ -4,6 +4,7 @@ image: quay.io/vgteam/vg_ci_prebake:latest
 variables:
   PYTHONIOENCODING: "utf-8"
   DEBIAN_FRONTEND: "noninteractive"
+  SET_OWNER_TAG: "toil-shared-ci@ucsc.edu"
```

Comment:
Can this address actually receive e-mail? We want all the owner tags to correspond to real people who are IAM users in the gateway and can receive e-mail. If it's a shared bucket it should be tagged with an Owner of `shared`, but that's most appropriate for long-lived buckets with many people's data.

Reply:
I wanted a "shared" tag that flags appropriately for billing, and I don't believe that this is specific to you or I, but anyone developing on Toil.  We should discuss this on Wednesday.

Reply:
We didn't manage to talk about this at the meeting today. Maybe this should just be `shared`?